bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/Ghost.scala

```diff
@@ -285,11 +314,43 @@ final class Ghost extends SPInstObsComp(GhostMixin.SP_TYPE) with PropertyProvide
   }
 
   override def getVignettableScienceArea: ScienceAreaGeometry = GhostScienceAreaGeometry
+
+  override def pwfs2VignettingClearance(ctx: ObsContext): Angle = {
```

Comment:
VIGNETTING

Here is the important part: I'm trying to find the target or sky position that's furthest away from the base. My idea is then we can add a bit more (I'm not sure if this is statically calculated or dependent on the `ObsContext`) to avoid vignetting the targets and the space positions. This is the code to find the furthest object. I think it can be improved upon, so any suggestions would be very helpful.